.github/workflows/release.yml

```diff
@@ -121,6 +121,8 @@ jobs:
             ${{ runner.OS }}-
       - name: Install Packages
         run: npm ci
+        env:
+          GITHUB_PACKAGE_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this actually required?

Reply:
seems not!

Reply:
actually I removed the cache and it fails without env